specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2016-05-15/DTL.json

```diff
@@ -58,7 +74,7 @@
             "schema": { "$ref": "#/definitions/CloudError" }
           }
         },
-        "security": [ { "oauth2": [ "user_impersonation" ] } ],
+        "security": [ { "azure_auth": [ "user_impersonation" ] } ],

```

Comment:
[Taking a look at the Redis swagger](https://github.com/Azure/azure-rest-api-specs/blob/b9062411a7ac5b5dc3a8840b5c5e0f8200e9a8d3/specification/redis/resource-manager/Microsoft.Cache/stable/2017-10-01/redis.json#L25), I believe that in addition to renaming the security line from `oauth2` to `azure_auth`, one must define a "securityDefintions" clause.

More info here from the spec: https://swagger.io/specification/#securitySchemeObject

Reply:
yeah, i already have that in line 8717

Reply:
Excellent, thanks for the link. Sorry I didn't see it!